raft-ann-bench/cpu/Dockerfile

```diff
@@ -0,0 +1,48 @@
+# syntax=docker/dockerfile:1
+
+ARG PYTHON_VER=3.10
+ARG RAPIDS_VER=23.10
+
+FROM condaforge/mambaforge:23.3.1-0 as raft-ann-bench-cpu
+ARG RAPIDS_VER
+ARG PYTHON_VER
+
+# # Create a conda group and assign it as root's primary group
+RUN groupadd conda; \
+  usermod -g conda root
+
+# # Ensure new files are created with group write access & setgid. See https://unix.stackexchange.com/a/12845
+# RUN chmod g+ws /opt/conda
+
+RUN useradd -rm -d /home/rapids -s /bin/bash -g conda -u 1001 rapids
+
+# USER rapids
+
+WORKDIR /home/rapids
+
+COPY condarc /opt/conda/.condarc
+
+RUN mamba install -y -n base \
+        "raft-ann-bench-cpu=${RAPIDS_VER}.*" \
+        "python=${PYTHON_VER}" \
+    && conda clean -afy
```

Comment:
Add a `mamba update --all -y -n base` before the clean to ensure latest versions of everything.